apps/cricket-score-frontend/src/components/match/NotFoundDisplay.tsx

```
@@ -0,0 +1,26 @@
+import React from 'react';
+import { motion } from 'framer-motion';
+
+export const NotFoundDisplay: React.FC = () => (
+  <div className="flex justify-center items-center min-h-screen bg-gradient-theme p-4">
+    <motion.div
+      initial={{ opacity: 0, y: 20 }}
+      animate={{ opacity: 1, y: 0 }}
+      className="w-full max-w-md text-center glass-card p-8"
+    >
+      <div className="w-16 h-16 mx-auto bg-muted/40 backdrop-blur-sm rounded-full flex items-center justify-center border-2 border-muted/60 relative shadow-xl">
+        <svg xmlns="http://www.w3.org/2000/svg" className="h-8 w-8 text-muted-foreground" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)